cwms-data-api/src/test/java/cwms/cda/data/dao/timeseriesprofile/TimeSeriesProfileDaoIT.java

```diff
@@ -0,0 +1,130 @@
+package cwms.cda.data.dao.timeseriesprofile;
+
+import java.sql.SQLException;
+import java.util.Arrays;
+import java.util.List;
+
+import cwms.cda.api.DataApiTestIT;
+import cwms.cda.data.dto.CwmsId;
+import cwms.cda.data.dto.timeseriesprofile.TimeSeriesProfile;
+import cwms.cda.data.dto.timeseriesprofile.TimeSeriesProfileTest;
+import fixtures.CwmsDataApiSetupCallback;
+import mil.army.usace.hec.test.database.CwmsDatabaseContainer;
+import org.jooq.DSLContext;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import static cwms.cda.data.dao.DaoTest.getDslContext;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@Tag("integration")
+class TimeSeriesProfileDaoIT extends DataApiTestIT {
+    @Test
+    void testCopyTimeSeriesProfile() throws SQLException {
+        CwmsDatabaseContainer<?> databaseLink = CwmsDataApiSetupCallback.getDatabaseLink();
+        databaseLink.connection(c -> {
```

Comment:
all of these usages need to have `CwmsDataApiSetupCallback.getWebUser()` added as the second parameter so the correct user with appropriate office permissions is used during the operations.

Reply:
added getWebUser as the second parameter throughout